frontend/public/components/utils/dropdown.jsx

```diff
@@ -92,8 +92,15 @@ const Caret = () => <span className="caret" />;
 
 class DropDownRow extends React.PureComponent {
   render() {
-    const {itemKey, content, onclick, onBookmark, onUnBookmark, className, selected, hover, canFavorite, onFavorite, favoriteKey} = this.props;
+    const {itemKey, content, onclick, onBookmark, onUnBookmark, className, selected, hover, canFavorite, onFavorite, favoriteKey, autocompleteFilter} = this.props;
     let prefix;
+
+    if(!autocompleteFilter){
```

Comment:
style nit, here and below

```suggestion
    if (!autocompleteFilter) {
```